server/testServer.js

```diff
@@ -1,17 +1,18 @@
-
 //This is strictly for testing purposes
```

Comment:
Philippe, is there any reason to keep this file?

Reply:
Looks like something we'd use for testing, I'd be fine with keeping this file in.

Reply:
We should be able to run all of our tests in app.js in development mode, plus it'll be a single place that we know everything works instead of needing to keep the two up to date.